src/UglyToad.PdfPig/Tokens/HexToken.cs

```diff
@@ -79,15 +78,29 @@ public HexToken([NotNull] IReadOnlyList<char> characters)
 
                 var b = Convert(high, low);
                 bytes.Add(b);
+            }
+
+            // Handle UTF-16BE format strings.
+            if (bytes.Count >= 2 && bytes[0] == 0xFE && bytes[1] == 0xFF)
+            {
+                Data = Encoding.BigEndianUnicode.GetString(bytes.ToArray(), 2, bytes.Count - 2);
+            }
+            else
```

Comment:
There's another case where the string uses LE Unicode starting with (hex) `FFFE` which it's probably worth adding as an extra `else if` here. I haven't seen it in the wild yet but I think it's worth covering:

```               
else if (bytes.Count >= 2 && bytes[0] == 0xFF && bytes[1] == 0xFE)
                {
                    Data = Encoding.Unicode.GetString(bytes.ToArray(), 2, bytes.Count - 2);
                }```

Reply:
I'd been attempting to read the pdf spec and wasn't sure if the LE case was needed here (the 'text string type'/ section 7.9.2.2 of the spec I was looking at only mentioned the BE case).
I can add that case later if you want, though I don't have any 'real' test data for it.

Reply:
Ah, fair enough, yeah it's not the clearest so I'll merge this fix in then in future it's easy to fix if the LE case does turn out to be needed.